### ISSUE TITLE
Make Readme file more useful for devs.

### ISSUE THREAD

**Issue by davmillar** [davmillar/DavesMapper#54]
Add info about starting up the environment, what tools are needed, where the meat of the code lives, etc.

**Comment by cancinos** [comment]
I'd like to help you! But just forking you repository won't give me all the information that I need!
wanna help me helping you? @davmillar

**Comment by davmillar** [comment]
@cancinos absolutely! I'm just not familiar with what is needed to most easily fork and build up a project, so please let me know.

**Comment by suniahk** [comment]
Some basic notes from trying to get the project up and running on my own (Windows) machine today.
Requirements:
- Vagrant
- Your Hypervisor of choice (I already had virtualbox installed)
- npm
- java
- yuicompressor.jar
- command line inkscape (I have it installed but not on my PATH so icons didn't properly build)

npm install -g *
- less

I think that was all it took to get up and running, although since I didn't have any images it makes feature testing difficult for anything involving the tiles.  I can write this up in the README.  That being said, the code itself would probably benefit from being split into smaller files; having "Mapper" and "mapping" makes things difficult to follow for someone working in the codebase for the first time.

**Comment by davmillar** [comment]
@suniahk thanks, I appreciate that feedback. The codebase definitely ought to be split up, and that’s something I’ve been working on but not quite finished yet. I’ll take some time this weekend to add better reference materials on that and update some of these issues.
